pages/omnibar.js

```diff
@@ -995,9 +995,11 @@ var SearchEngine = (function() {
         return this.activeTab;
     };
     self.onInput = function() {
-        if (!self.suggestionURL || typeof(self.listSuggestion) !== "function") {
-            return;
-        }
+        const canSuggest = self.suggestionURL && typeof(self.listSuggestion) === "function"
```

Comment:
Great, thanks for the fix. 👍 

Could you please also add semicolon at end of these two lines?